GameDefinition/Content/ItemKind.hs

```diff
@@ -133,7 +133,7 @@ spike2 = spike
                , SetFlag MinorEffects
                , Odds (10 * 1 `dL` 10) [] [toVelocity 70] ]
                    -- at deep levels sometimes even don't limit velocity
-  -- , idesc    = ""
+  , idesc    = "A jagged spur of metal."
```

Comment:
Notice `spike2` is a very rare more damaging version of `spike2`. So I'd go for something more dramatic or perhaps just exchange the two descriptions (though yours is "jagged" and the other is "small" so perhaps they are somhow matched in cruelty. Feel free to edit both.

Reply:
I've changed it to "a jagged skewer of rusty metal" - is that threatening enough?

Reply:
I'm scared. :)